### PR TITLE
Mccalluc/all assay types

### PR DESCRIPTION
**Pull request by mccalluc**

### CODE REVIEW

File: data/definitions/enums/assay_types.yaml [hunk 1]
```diff
@@ -47,4 +47,52 @@ seqFish_lab_processed:
...
+salmon_rnaseq_sciseq:
+    description: salmon_rnaseq_sciseq
+salmon_rnaseq_snareseq:
+    description: salmon_rnaseq_snareseq
+sc_atac_seq_snare:
+    description: sc_atac_seq_snare
+seqFISH:
+    description: seqFISH
+seqFISH_lab_processed:
+    description: seqFISH_lab_processed
```
Comment (ngehlenborg) [inline data/definitions/enums/assay_types.yaml:98]:
```suggestion
    description: seqFISH [Lab Processed]
```

File: data/definitions/enums/assay_types.yaml [hunk 2]
```diff
@@ -47,4 +47,52 @@ seqFish_lab_processed:
...
+dt_other:
+    description: dt_other
+dt_scatacdeq:
+    description: dt_scatacdeq
+dt_sciex:
+    description: dt_sciex
+dt_tmtlcms:
+    description: dt_tmtlcms
+image_pyramid:
+    description: image_pyramid
```
Comment (ngehlenborg) [inline data/definitions/enums/assay_types.yaml:86]:
```suggestion
    description: Image Pyramid
```

File: data/definitions/enums/assay_types.yaml [hunk 3]
```diff
@@ -47,4 +47,52 @@ seqFish_lab_processed:
...
+salmon_rnaseq_bulk:
+    description: salmon_rnaseq_bulk
```
Comment (ngehlenborg) [inline data/definitions/enums/assay_types.yaml:88]:
```suggestion
    description: Bulk RNA-seq [Salmon]
```

**Comment by mccalluc** [comment]
Ivan is going to to update this list, with just the codes they are actually using, so I'll close this.

**Pull request closed without merging**